playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -132,6 +132,7 @@ EDXAPP_CAS_ATTRIBUTE_PACKAGE: ""
 EDXAPP_ENABLE_AUTO_AUTH: false
 # Settings for enabling and configuring third party authorization
 EDXAPP_ENABLE_THIRD_PARTY_AUTH: false
+EDXAPP_ENABLE_OAUTH2_PROVIDER: false
```

Comment:
You mean this was recently removed and you are putting it back in? Can you add a comment to explain why this needs to be disabled by default in sandboxes?